package.json

```diff
@@ -17,14 +17,28 @@
     "executablePath": {
       "default": "tidy",
       "title": "Full path to the `tidy` executable",
+      "order": 1,
       "type": "string"
     },
+    "executableArguments": {
+      "default": [
+        "-utf8"
+      ],
+      "title": "Tidy Executable Arguments",
+      "description": "Additional arguments to pass to the Tidy executable when invoked.<br/><br/>The arguments specified here will be appended to arguments required for this linter to work.<br/><br/>Note that this option is an array. Each executable argument should be an element in the array.",
```

Comment:
How about "Comma-separated list of additional arguments..." and cutting out the last sentence? Most users aren't going to know what "this setting is an array" means.